File: Code Chef/JavaScript/200-299/213 Lucky Seven LUCKYSEVEN.js
//Problem Simplification:
    //Chef sees 10 letter word everyday.
    //He considers 7th letter as his lucky letter.

//Logic:
    //Set Encoding.
    //Create event listener that listens to data.
    //We'll have a single 10 character string as input.
    //Print 6th Index (counting from 0).

process.stdin.setEncoding("utf-8");
process.stdin.on("data",(input) => {
    console.log(input[6]);
});

File: Code Chef/C Programming/200-299/213 Lucky Seven LUCKYSEVEN.c
//Problem Simplification:
    //7 is lucky number of Chef
    //Chef sees 7th letter of a word
    //Word is having length of 10

//Logic:
    //create string array of 10 length
    //Take input of string
    //print 6th index because indexing starts with 0

#include <stdio.h>
int main(void)
{
    char s[10];
    scanf("%s",s);
    printf("%c",s[6]);
	return 0;
}